lib/datasource/pod/index.ts

```diff
@@ -98,26 +119,54 @@ const githubRegex = regEx(
   /^https:\/\/github\.com\/(?<account>[^/]+)\/(?<repo>[^/]+?)(\.git|\/.*)?$/
 );
 
+const githubEnterpriseRegex = regEx(
+  /(?<hostURL>(^https:\/\/github\..+\.[a-zA-z0-9-]+))\/(?<account>[^/]+)\/(?<repo>[^/]+?)(\.git|\/.*)?$/
+);
+
 async function getReleasesFromGithub(
   lookupName: string,
-  registryUrl: string,
-  useShard = false
+  opts: { hostURL: string; account: string; repo: string },
+  useShard = true,
+  useSpecs = true,
+  urtFormatOptions = URLFormatOptions.WithSharedWithSpec
```

Comment:
is this somehow configurable? Or are you only using the default value? Are you planning more changes?

Reply:
the `getReleasesFromGithub` method tries all the different url format using this configuration and I am using this as the default value. I am not planning to add more changes but in future there could be another format someone else may require.